src/sonic-frr/patch/series

```diff
@@ -6,30 +6,12 @@
 0006-Link-local-scope-was-not-set-while-binding-socket-for-bgp-ipv6-link-local-neighbors.patch
 0007-ignore-route-from-default-table.patch
 0008-Use-vrf_id-for-vrf-not-tabled_id.patch
-0009-bgpd-Ensure-suppress-fib-pending-works-with-network-.patch
 0010-bgpd-Change-log-level-for-graceful-restart-events.patch
-0011-zebra-Static-routes-async-notification-do-not-need-t.patch
-0012-zebra-Rename-vrf_lookup_by_tableid-to-zebra_vrf_look.patch
-0013-zebra-Move-protodown_r_bit-to-a-better-spot.patch
-0014-zebra-Remove-unused-dplane_intf_delete.patch
-0015-zebra-Remove-unused-add-variable.patch
-0016-zebra-Remove-duplicate-function-for-netlink-interfac.patch
-0017-zebra-Add-code-to-get-set-interface-to-pass-up-from-.patch
-0018-zebra-Use-zebra-dplane-for-RTM-link-and-addr.patch
-0019-zebra-remove-duplicated-nexthops-when-sending-fpm-msg.patch
-0020-zebra-Fix-non-notification-of-better-admin-won.patch
 0021-Disable-ipv6-src-address-test-in-pceplib.patch
 0022-cross-compile-changes.patch
-0023-zebra-The-dplane_fpm_nl-return-path-leaks-memory.patch
-0024-lib-use-snmp-s-large-fd-sets-for-agentx.patch
 0025-bgp-community-memory-leak-fix.patch
-0026-bgp-fib-suppress-announce-fix.patch
-0027-lib-Do-not-convert-EVPN-prefixes-into-IPv4-IPv6-if-n.patch
 0028-zebra-fix-parse-attr-problems-for-encap.patch
-0029-zebra-nhg-fix-on-intf-up.patch
 0030-zebra-backpressure-Zebra-push-back-on-Buffer-Stream-.patch
-0031-bgpd-backpressure-Add-a-typesafe-list-for-Zebra-Anno.patch
-0032-bgpd-fix-flushing-ipv6-flowspec-entries-when-peering.patch
```

Comment:
Please do git rm of all the patches that are removed. Those should be in deleted files.

Reply:
Done